src/utils/email.js

```diff
@@ -11,6 +12,7 @@ export function isEmail (s) {
 }
 
 const emailLikeRegex = /"([^"]+(@|%40)[^"]+[.][a-z]*(\s+)?)(\\"|")/
+export const emailLikeNoDoubleQuotesRegex = /([a-zA-Z0-9._-]+@[a-zA-Z0-9._-]+\.[a-zA-Z0-9_-]+)/
```

Comment:
I think [a-zA-Z-0-9] is too restrictive since it matches ASCII only and therefore would miss many valid emails. Usually (.+)@(\S+) is used. Maybe we can go with something like (?i)(?:mailto:)?(.+)@(\S+)? Not sure if modifiers and non-capturing groups are supported here, though.

Reply:
It makes sense to broaden the matching capability of the regex, but we should try matching emails anchored but the `mailto:` and also anchored by let's say, empty spaces, right?